### ISSUE TITLE
Observing model property on VM not working anymore

### ISSUE THREAD

**Issue by stefan505** [aurelia/binding#439]
A while back the following scenario would work, but has since stopped working:

Property inside a VM:
```
user: IUser;
```

With `IUser` as:

```
export interface IUser
{
    username: string;
    ...
}
```

Using `ObserverLocator` to raise property changes on `user.username` inside my VM is not working, `changed(newValue: any, oldValue?: any)` never being executed: 

```
_observer.getObserver(this, 'user.username').subscribe(this.changed);
```

Has anything changed?

**Comment by EisenbergEffect** [comment]
This API changed about 7 months ago I think. @jdanyow can advise you on how to use the new API.

**Comment by jdanyow** [comment]
Are you sure this worked: `getObserver(this, 'user.username')`?  This part of the API has never changed- it's always been `getObserver(object, propertyName)`.  Try `getObserver(this.user, 'username')`.

**Comment by stefan505** [comment]
@jdanyow yes, I used a multi observer for instance that wraps observable locator (inspired by [your blog post](https://www.danyow.net/aurelia-property-observation)). Actual code:

```
multiObserver.observe(this, ['user.EmailAddress', 'user.Username'], this.call);
```

I tried re-implementing the same multi-observer on all the latest bits without luck. The above code worked with `aurelia-binding@1.0.0-beta.2.0.7`, which is what my `config.js` was pegged at.

`getObserver(this.user, 'username')` does work and I've implemented it as such.
